M3T1/BLL/CartService.cs

```diff
@@ -0,0 +1,28 @@
+﻿public class CartService : ICartService
+{
+    private readonly ICartRepository _repository;
+
+    public CartService(ICartRepository repository) => _repository = repository;
+
+    public Task<List<CartEntity>> GetAll() => _repository.GetByFilter();
+
+    public async Task AddItemFromCartById(int id)
```

Comment:
How do you add details of the cart item?

Reply:
Also, what does mean "AddItemFromCartById"? As I can see in requirements we should add Item to Cart